cli/popper/__init__.py

```diff
@@ -1 +1,21 @@
-__version__ = "2.5.1-dev"
+import importlib
+import os
+
+# check if dunamai is available
+dunamai_spec = importlib.util.find_spec("dunamai")
+dunamai_found = dunamai_spec is not None
+if dunamai_found:
+    # if dunamai is found, then we use it to display the version
+    import dunamai
+
+    __version__ = dunamai.Version.from_any_vcs().serialize()
+    ver = f'__popper_version__ = "{__version__}"'
+    path = os.path.split(os.getcwd())
```

Comment:
since `popper version` might not be executed from the popper repository folder always, the value returned by `os.getcwd()` doesn't necessarily correspond to the project root directory. Instead, what we can do is obtain the path to the current script, which in python 3 can be done by doing:

```python
import pathlib
_init_script_dir = pathlib.Path(__file__).parent.absolute()
```

So the `_init_script_dir` will always point to the folder containing the `__init__.py` script.